components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ThumbnailLoader.java

```diff
@@ -380,19 +380,21 @@ private ThumbnailStorePrx getThumbnailStore(PixelsData pxd) throws DSAccessExcep
     }
 
     /**
-     * Returns whether a pyramid should be used for the given {@link PixelsData}.
-     * This usually implies that this is a "Big image" and therefore will need
+     * Returns whether a pyramid should be used for the given {@link PixelsData}
+     * . This usually implies that this is a "Big image" and therefore will need
      * tiling.
      *
      * @param pxd
      * @return
      */
     private boolean requiresPixelsPyramid(PixelsData pxd) throws ServerError,
             DSOutOfServiceException {
-        int maxWidth = context.lookup(LookupNames.MAX_PLANE_WIDTH) != null ? (Integer) context
-                .lookup(LookupNames.MAX_PLANE_WIDTH) : 3192;
-        int maxHeight = context.lookup(LookupNames.MAX_PLANE_HEIGHT) != null ? (Integer) context
-                .lookup(LookupNames.MAX_PLANE_HEIGHT) : 3192;
+        int maxWidth = context.lookup(LookupNames.MAX_PLANE_WIDTH) != null ? Integer
+                .parseInt((String) context.lookup(LookupNames.MAX_PLANE_WIDTH))
+                : 3192;
+        int maxHeight = context.lookup(LookupNames.MAX_PLANE_HEIGHT) != null ? Integer
```

Comment:
Should actually 'bind' an Integer in first place instead of doing the conversion here, will fix that in another commit.